prow/cmd/phaino/README.md

```diff
@@ -41,6 +41,37 @@ volumes that the Prow Job may require.
 * `--use-local-gcloud-credentials` controls whether to use the same gcloud credentials as local or not
 * `--use-local-kubeconfig` controls whether to use the same kubeconfig as local or not
 
+#### Common options usage scenarios
+
+Phaino is smart at prompting for where repo is located, volume mounts etc., if
+it's desired to save the prompts, use the following tricks instead:
+
+- If the repo is not cloned under GOPATH, use:
+  ```
+  --gopath=/whatever/go/src # Controls where source code is cloned in container
+  --extra-volume-mounts=/whatever/go/src/k8s.io/test-infra=/Users/xyz/k8s-test-infra
+  ```
+- If job requires mounting kubeconfig, assume the mount is named `kubeconfig`,use:
+  ```
+  --use-local-kubeconfig
+  --skip-volume-mounts=kubeconfig
+  ```
+- If job requires mounting gcloud default credentials, assume the mount is named `service-account`,use:
+  ```
+  --use-local-gcloud-credentials
+  --skip-volume-mounts=service-account
+  ```
+- If job requires mounting something else like `name:foo; mountPath: /bar`,use:
```

Comment:
Hmm is this example correct?

Reply:
yes I believe so, it's not very intuitive to me tbh